package on2021_06.on2021_06_10_Codeforces_Round__725__Div__3_.D___Another_Problem_About_Dividing_Numbers;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2021_06/on2021_06_10_Codeforces_Round__725__Div__3_/D___Another_Problem_About_Dividing_Numbers/D - Another Problem About Dividing Numbers.json"))
			Assert.fail();
	}
}
